README.md

```diff
@@ -1,5 +1,7 @@
 # Create GitHub App Token
 
+[![test](https://github.com/actions/create-github-app-token/actions/workflows/test.yml/badge.svg)](https://github.com/actions/create-github-app-token/actions/workflows/test.yml) [![package-check](https://github.com/actions/create-github-app-token/actions/workflows/package-check.yml/badge.svg)](https://github.com/actions/create-github-app-token/actions/workflows/package-check.yml) ![Coverage](badges/coverage.svg)
```

Comment:
I'm not a big fan of all these badges 🤷🏼 I'd add the test one and leave it at that.

Reply:
Done!